load_data.js

```diff
@@ -0,0 +1,63 @@
+(function(window){
+  window.extractData = function() {
+    var ret = $.Deferred();
+
+    FHIR.oauth2.ready(function(smart){
+      var patient = smart.patient;
+      var pt = patient.read();
+     
+      $.when(pt).done(function(patient){

```

Comment:
Something you might want to add is logic for if the patient call fails, or if the FHIR Oauth2 call fails. We want to display some appropriately-described error messages on the UI side to let the user know why the app won't load.